chapters/technology-research/framework.tex

```diff
@@ -0,0 +1,315 @@
+\section{Multiplatformní framework}
+
+Vývoj mobilních aplikací je dle \cite{wepc_video_game_statistics} na vzestupu.
+To přináší nároky na vydavatele a vývojáře k rychlejší tvorbě mobilních
+aplikací a her. 
+Aplikace pro mobilní zařízení se běžně vytvářejí pomocí programovacích jazyků,
+které jsou úzce spjaty s mobilními zařízeními.
+Příkladem takového programovacího jazyka je jazyk Java,
+který se použivá pro vývoj nativních aplikací v platformě Android.
+Tyto jazyky mají úzkou podporu pro vývoj na daných zařízeních,
+včetně přístupu k hardwarovému rozhraní (kamera, senzory, \dots{}),
+jak uvádí zdroj \cite{dashmagazine_mobile_frameworks}. 
+
+\begin{figure}[ht!]
+    \centering
+    \includegraphics[width=\linewidth]{assets/technology-research/framework/platform_sdk.png}
+    \caption{Schéma frameworku specifického pro platformu \todo{do vektoru} \cite{hackernoon_flutter}}
+    \label{fig:framework_platform}
+\end{figure}
+
+Snahu urychlit vývoj mobilních aplikací řeší také mobilní frameworky.
+Tyto frameworky jsou tvořeny většinou stejnou společností,
+která také vytváří danou platformu
+a jsou publikování jako \emph{Software Development Kit} (dále jen SDK).
+\cite{dashmagazine_mobile_frameworks}
```

Comment:
Zde není jednoznačné k čemu citace patří. Značka by měla být přímo ve větě - před tečkou, případně za odstavcem (méně preferované, ale vhodné pokud by bylo ve většině vět odstavce). Takto je to na více místech v práci.

Jinak je ještě vhodné používat nedělitelnou mezeru před citací `...~\cite{...}` (nevím jestli to vlna řeší).